docs/getting-started/signing-transactions.md

```diff
@@ -3,86 +3,227 @@ sidebar_position: 3
+Firstly, we'll install json-rpc-tools and browserify with npm:
+```sh
+npm i @json-rpc-tools/utils browserify
+```
+Then, we create a file `main.js`:
+```js
+var jsonRpcTools = require("@json-rpc-tools/utils");
+global.window.jsonRpcTools = jsonRpcTools
+```
+Then, the following command is run to convert the library into a browser compatible module
+```sh
+browserify main.js -o bundle.js
+```
+Once run, it will be possible to import the `json-rpc-tools` library as a script:
+```html
+<script src="/bundle.js"></script>
 ```
 
-## Translate a doc
+### Connecting to the wallet
+To connect to a wallet, we will use the following function:
+```js
+async function setUpWallet() {
+    const wallet = new WalletConnect.default({
+        bridge: "https://bridge.walletconnect.org",
+        qrcodeModal: WalletConnectQRCodeModal.default,
+    });
+    if (!wallet.connected) {
+        wallet.createSession();
+        await new Promise((resolve) => {
+            wallet.on("connect", (error, payload) => {
+                if (error) { throw error; }
+                resolve();
+            });
+        });
+    }
+    return wallet;
+}
+```
+Let us analyze it line by line. On the first four lines, an instance of WalletConnect is created, which will be used for all consequent interactions with the wallet.
+```js
+const wallet = new WalletConnect.default({
+    bridge: "https://bridge.walletconnect.org",
+    qrcodeModal: WalletConnectQRCodeModal.default,
+});
+```
+If the user is not yet connected to its wallet, then `wallet.createSession()` is run, which prompts the user to scan a QR code. The function will then wait for the user to connect before returning.
+```js
+if (!wallet.connected) {
+    wallet.createSession();
+    await new Promise((resolve) => {
+        wallet.on("connect", (error, payload) => {
+            if (error) { throw error; }
+            resolve();
+        });
+    });
+}
+```
+In the meanwhile, the user will be able to open up the Algorand Wallet app, and scan the QR code
 
-Copy the `docs/intro.md` file to the `i18n/fr` folder:
+<img src="/img/algorandwallet_home.jpg" height="500px"/>
+<img src="/img/algorandwallet_scan.jpg" height="500px"/>
+<img src="/img/algorandwallet_connect.jpg" height="500px"/>
 
-```bash
-mkdir -p i18n/fr/docusaurus-plugin-content-docs/current/
+### Getting the account address
+Now that we have an instance of WalletConnect open, it is possible to get the connected account by running
+```js
+wallet.accounts[0]
+```
 
-cp docs/intro.md i18n/fr/docusaurus-plugin-content-docs/current/intro.md
+### Signing the transactions
+The following snippet creates an array of objects, each containing the transaction itself and, for the transactions which should not be signed, an empty array of signers. 
+```js
+const group = noSign.map(txn => ({
```

Comment:
Is it required by a wallet standard to supply the entire transaction group?

Reply:
Yep, usually the whole group is needed, otherwise you don't really know what you're signing. MyAlgoWallet for some reason behaves differently though.